src/Microsoft.AspNetCore.OData/Batch/ODataBatchHttpRequestMessageExtensions.cs

```diff
@@ -52,7 +54,8 @@ public static bool IsODataBatchRequest(this HttpRequest request)
                 throw Error.ArgumentNull("request");
             }
 
-            return request.ContentType?.StartsWith("multipart/", StringComparison.OrdinalIgnoreCase) ?? false;
+            return (request.ContentType != null &&
+                (request.ContentType.StartsWith(BatchMediaTypeMime) || request.ContentType.StartsWith(BatchMediaTypeJson)));
```

Comment:
>StartsWith [](start = 91, length = 10)

same here?

Reply:
same as above.

---
In reply to: [202867672](https://github.com/OData/WebApi/pull/1536#discussion_r202867672) [](ancestors = 202867672)